rd-kt/rd-framework/src/test/kotlin/com/jetbrains/rd/framework/test/util/TestBase.kt

```diff
@@ -35,6 +35,7 @@ open class TestBase {
     }
 
     @AfterEach
+    @Timeout(value = 30, unit = TimeUnit.SECONDS)
```

Comment:
Why do we need this, isn't the whole `TestBase` already covered by the identical `@Timeout`?

Turns out: nope. As documented by `@Timeout`, test lifecycle methods (such as `@AfterEach`) aren't covered by the test-wide `@Timeout`. So, we'll have to add one here.